R/ppm.R

```diff
@@ -201,8 +201,9 @@ renv_ppm_platform_rhel <- function(properties) {
   id <- properties$VERSION_ID
   if (is.null(id))
     return(NULL)
+  rhel_version <- ifelse(as.integer(id) < 9, "centos", "rhel")
```

Comment:
Nit: we should use `numeric_version()` here instead of `as.integer()` in case the version here happened to be something like `9.10` in some theoretical future.